src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import './UserInfo.scss';
+import { Users } from '../../types/Users';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  user: Users;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const { email, name } = user;
+
```

Comment:
you can destruct in props
```suggestion
export const UserInfo: React.FC<Props> = ({ user: { email, name }}) => {
```